Makefile

```diff
@@ -63,3 +65,24 @@ lint: fmt
 .PHONY: fmt
 fmt: install-tools
 	gofumpt -l -w -extra  .
+
+cleanup-warehouse-integration:
+	docker-compose -f warehouse/docker-compose.test.yml down --remove-orphans --volumes
+
+setup-warehouse-integration: cleanup-warehouse-integration
+	docker-compose -f warehouse/docker-compose.test.yml up --build start_warehouse_integration
+
+logs-warehouse-integration:
+	docker logs wh-backend
+
+run-warehouse-integration: setup-warehouse-integration
+	if docker-compose -f warehouse/docker-compose.test.yml exec -T wh-backend go test -v ./warehouse/... -tags=warehouse_integration -timeout 30m -count 1; then \
```

Comment:
Verify if -p factor is working.

Reply:
After removing it kind increased the test time. Currently keeping this value as 8.